plugin.yml

```diff
@@ -1,10 +1,10 @@
 name: Slapper
 author: jojoe77777
-version: 1.5.3
+version: 1.5.4
 description: Slapper, the NPC plugin for PocketMine-MP
 main: slapper\Main
-api: 3.13.0
-mcpe-protocol: [361, 388, 389, 390]
+api: 3.14.0
```

Comment:
this is unnecessary

Reply:
There are a few changes that need to be made in order for slapper to function properly with MCPE 1.16's new skin bullshit. Do we want to support MP 3.14 as well?

Reply:
To the best of my knowledge slapper doesn't need any changes at all beyond a protocol bump...

Reply:
Ah ok, so the skin mess is something with the PM core? If you're not already aware of the issue, skin data is not being saved/loaded correctly on the human entities spawned by Slapper.

Reply:
> Ah ok, so the skin mess is something with the PM core? If you're not already aware of the issue, skin data is not being saved/loaded correctly on the human entities spawned by Slapper.

pocketmine\network\mcpe\protocol\AddActorPacket::$type
So this is what its creating a Steve npc instead of ur skin?

Reply:
the steve skins are caused by the "only allow trusted skins" setting in the client.

Reply:
Is there a way to not only allow trusted skins??

Reply:
There's probably a way to force the client setting